src/main/resources/db/changelog/changes/08-create-book-category-table.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-book-category-table
+      author: Dmytro Varukha
+      changes:
+        - createTable:
+            tableName: book_category
+            columns:
+              - column:
+                  name: book_id
```

Comment:
add required constraints to for this table:
- primary key (book_id, category_id)
- fk (book_id)
- fk (category_id)